extras/indexing/src/main/java/mvm/rya/indexing/accumulo/geo/GeoMesaGeoIndexer.java

```diff
@@ -435,8 +435,11 @@ public void close() throws IOException {
 
     @Override
     public String getTableName() {
-       return ConfigUtils.getGeoTablename(conf);
+            return getTableName(conf);
     }
+    public static String getTableName(Configuration conf) {
```

Comment:
For all of these types of methods, why not just require the table prefix as a parameter instead of the Configuration object? It would be a simpler method that way.